modules/gapi/misc/python/test/test_gapi_imgproc.py

```diff
@@ -75,5 +75,27 @@ def test_rgb2gray(self):
                              'Failed on ' + pkg_name + ' backend')
 
 
+    def test_bounding_rect(self):
+        sz = 1280
+        fscale = 256
+
+        def sample_value(fscale):
+            return np.random.uniform(0, 255 * fscale) / fscale
+
+        points = np.array([(sample_value(fscale), sample_value(fscale)) for _ in range(1280)], np.float32)
+
+        # OpenCV
+        expected = cv.boundingRect(points)
+
+        # G-API
+        g_in  = cv.GMat()
+        g_out = cv.gapi.boundingRect(g_in)
```

Comment:
So does this work without defining `GOpaque<Size>` as `GSize` ?

Reply:
No, because it's needed to wrap `GOpaque<Size>` into python somehow

Reply:
Not relevant
use `cv.GOpaqueT(cv.gapi.CV_SIZE)`